packages/appstore/src/components/options/options.scss

```diff
@@ -0,0 +1,171 @@
+.options-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1.5rem;
+    width: 115.2rem;
+    height: 30.3rem;
+    @include mobile {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        position: absolute;
+        width: 32.8rem;
+        height: 28rem;
+    }
+
+    &__title-description-container {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        width: 115.2rem;
+        height: 6rem;
+        @include mobile {
+            width: 32.8rem;
+            height: 5.4rem;
+            font-family: 'IBM Plex Sans';
```

Comment:
Our default font family is IBM Plex sans (As far as I am aware of). So you do not need to set it. it is redundant here. You would face some issues with the tests in the github (I had the same issue when used your component)